.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
 **__pycache__/
 *.DS_Store
-.coverage
\ No newline at end of file
+.coverage
+src/orgwarden/test.py
```

Comment:
What is `test.py` and why is it being ignored? Is this something left over from a previous commit?